src/main/java/org/commcare/cases/query/QuerySensitive.java

```diff
@@ -0,0 +1,9 @@
+package org.commcare.cases.query;
+
+/**
+ * Created by ctsims on 1/27/2017.
+ */
+
+public interface QuerySensitive {
```

Comment:
That's a fair concern. My worry was that Will might need to implement the "poke" concept on some Web apps specific code, which would require making core changes to the libraries in order to proceed. If the ability to sign up for a poke comes from your own object it lets him (or android) accept that signal without needing a new build of the core libraries.